collector/project.go

```diff
@@ -106,6 +121,42 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 			p.Pod.Update(totalPo)
 			poUtils = append(poUtils, float64(totalPo))
 		}
+
+		// Apps
+		if len(globalRancherLibraryNames) > 0 {
+			duplicateLibraryNames := false
+			for _, catalogName := range globalRancherLibraryNames {
+				// Make sure rancher library names aren't duped in this project or cluster
+				pCatalog, err := c.Client.ProjectCatalog.ByID(
+					fmt.Sprintf("%s:%s", projectID, catalogName))
+				if err == nil && pCatalog != nil {
```

Comment:
What if the catalog is a dupe and has the same url as the global one?

Reply:
Could add another check here for that I guess. Really this whole thing isn't a total solution, just a way to get majority of setups.